src/actions/receive-profile.js

```diff
@@ -1109,10 +1071,17 @@ export function retrieveProfileFromBrowser(): ThunkAction<Promise<void>> {
           );
         }, 30000);
         geckoProfiler = await window.geckoProfilerPromise;
+        rawGeckoProfile = await geckoProfiler.getProfile();
```

Comment:
Accrording to a removed comment ` // This fallback can be removed once the oldest supported Firefox ESR version is 93 or newer.` what do you think about removing most of the old code? Unless you wanted to do it in another PR? If that's your plan, please add the comment back or update the comment above in this function.

Reply:
Those comments are wrong, I'll need to do another pass over the code to fix the comments, once https://bugzilla.mozilla.org/show_bug.cgi?id=1625309 lands.

Reply:
It'll land in 96 at the earliest, so all mentions of 93 actually need to refer  to 96.

Reply:
oh right, I confusely forgot about it.